host/activity_test.go

```diff
@@ -1063,7 +1063,11 @@ func (s *integrationSuite) TestActivityHeartbeatTimeouts() {
 	for i := 0; i < activityCount; i++ {
 		go func() {
 			err := poller.PollAndProcessActivityTask(false)
-			s.Logger.Error("Activity Processing Completed", tag.Error(err))
+			if err == nil {
+				s.Logger.Error("Activity expected to time out but got completed instead")
+			} else {
+				s.Logger.Info("Activity processing completed with error", tag.Error(err))
```

Comment:
just thinking out loud, i wonder if it is easy / worth it to detect that the error is a timeout (which is expected), and, if so, make that clear in the log message that the failure is expected, something along the lines of 
```golang
if err == nil {
    s.Logger.Error("omg this didn't timeout, unexpected!)
    // maybe fail the test
} else if is_a_timeout(err) {
    s.Logger.Info("very legal very cool, timed out, as expected")
} else {
   s.Logger.Error("omg failed in unexpected ways!")
   // fail the test
}
```